src/internal/functions/Get-ReadmeContent.ps1

```diff
@@ -42,7 +42,7 @@ function Get-ReadmeContent {
   Begin {
     $BuilderModuleGalleryUri      = 'https://www.powershellgallery.com/packages/puppet.dsc'
     $BuilderModuleRepository      = 'https://github.com/puppetlabs/Puppet.Dsc'
-    $BaseProviderSource           = 'https://github.com/puppetlabs/ruby-pwsh/blob/master/lib/puppet/provider/dsc_base_provider.rb'
+    $BaseProviderSource           = 'https://github.com/puppetlabs/ruby-pwsh/blob/main/lib/puppet/provider/dsc_base_provider.rb'
```

Comment:
This url doesn't exist but this one does: https://github.com/puppetlabs/ruby-pwsh/blob/main/lib/puppet/provider/dsc_base_provider/dsc_base_provider.rb

Maybe get input from @michaeltlombardi

Reply:
Have updated, Will check with @michaeltlombardi when he gets online

Reply:
Awesome catch, y'all!